### PR TITLE
chore(#8957): translate to spanish 900 to 1200

### PR DESCRIPTION
**Pull request by latin-panda**
# Description

Translation to Spanish from L900 to L1200
#8957

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs
If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:Translate-to-Spanish-900-to-1200/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:Translate-to-Spanish-900-to-1200/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:Translate-to-Spanish-900-to-1200/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Review by lorerod — CHANGES_REQUESTED** [review]

File: api/resources/translations/messages-es.properties [hunk 1]
```diff
@@ -1146,50 +1151,59 @@ report.immunization_visit.vaccines_received.received_pneumococcal_1 = Neumococo
...
-report.pregnancy.danger_signs = Signos de alerta
-report.pregnancy.edd =
-report.pregnancy.lmp_date = Fecha LMP
-report.pregnancy.patient_id =
+report.pregnancy.chw_name = Nombre del Trabajador de Salud Comunitario
+report.pregnancy.chw_phone = Número de Teléfono del Trabajador de Salud Comunitario
+report.pregnancy.chw_sms = Notas al Trabajador de Salud Comunitario
+report.pregnancy.danger_signs = Señales de Peligro
+report.pregnancy.edd = FEP
+report.pregnancy.lmp_date = Fecha FUM
```
[the remark this answers is not in the document]
Comment (latin-panda) [reply api/resources/translations/messages-es.properties:1186]: Here I changed it to: Fecha de última menstruación

File: api/resources/translations/messages-es.properties [hunk 2]
```diff
@@ -898,241 +898,246 @@ messages.schedule.child.month_26 = Visite {{patient_name}} {{patient_id}} para a
...
+partner.logo.field = Logo del socio
+partner.logo.upload = Subir logo del socio
+partner.name.field = Nombre del socio
+partner.supporting = Socios que está apoyando
+partner.tab.partners = Socios
+password.incorrect = La contraseña no es correcta.
+password.length.minimum = La contraseña debe tener al menos {{minimum}} caracteres.
+password.update = Actualizar contraseña
+password.weak = La contraseña es demasiado fácil de adivinar. Incluya más variedad de caracteres para hacerlo más complejo.
+patient\ id\ not\ found\ response = Envíe el siguiente mensaje de respuesta las validaciones pasan correctamente pero no hay ningún valor para el campo de Medic ID.
```
[the remark this answers is not in the document]
Comment (latin-panda) [reply api/resources/translations/messages-es.properties:977]: I missed some words. The English is: `Send the following response message if the validations pass but the Medic ID is not located.`
My understanding is: The message will be sent if 
- medic id is not found
- and all other validation passes

I changed it to: `Enviar el siguiente mensaje de respuesta, sí las validaciones pasan correctamente pero no se encontró el Medic ID.`
What do you think?

**Comment by latin-panda** [comment]
Thanks @lorerod! I've resolved the feedback

**Review by lorerod — APPROVED** [review]
Thank you @latin-panda!

**Pull request merged**